src/tiktokapipy/api.py

```diff
@@ -150,6 +151,7 @@ def __init__(
             specify the name of the dump file (exluding '.json').
         :param emulate_mobile: Whether to emulate a mobile device during sraping. Required for retrieving data
             on slideshows.
+        :param navigator_type: Whether to launch Playwright with 'Chromium' or 'Firefox'. Set to 'Chromium' as default.
```

Comment:
Minor thing, the documentation actually automatically marks the default value, so the "Set to 'Chromium' as default" is a bit redundant.

Reply:
If you want, I can re-upload without that redundancy.

Reply:
I just pushed a new version with that change